api/queries_pr.go

```diff
@@ -944,6 +944,24 @@ func PullRequestClose(client *Client, repo ghrepo.Interface, pr *PullRequest) er
 	return err
 }
 
+func PullRequestComment(client *Client, repo ghrepo.Interface, pr *PullRequest) error {
+	var mutation struct {
+		AddComment struct {
+			ClientMutationID string
+		} `graphql:"addComment(input: $input)"`
+	}
+
+	input := githubv4.AddCommentInput{
+		SubjectID: pr.ID,
+		Body:      githubv4.String(pr.Body),
+	}
+
+	v4 := githubv4.NewClient(client.http)
+	err := v4.Mutate(context.Background(), &mutation, input, nil)
```

Comment:
This is great, I'm glad you used the githubv4 pattern instead of our old string-based graphql pattern!